src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -134,6 +134,16 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
         } catch (err) {
             return "Input must be a number"
         }
-        return null;
+        return undefined;
+    }
+
+    private static validateCollectionName(name: string): string | undefined | null {
+        if (name.endsWith(" ")) {
```

Comment:
if (!name) ...

I've seen cases where we can get called with an undefined value and that would throw.